pull_request_template.md

```diff
@@ -0,0 +1,3 @@
+# This is a Teamwork passion project and not officially supported by the team
+
+The creators and maintainers of this project are doing it with enthusiasm and happiness, in odd moments between other projects and possibly even in their spare time. Please be kind to them, and understand that although they hope to get the time & energy to fix your bug :crossed_fingers:, this is not an officially supported product or service, and therefore doing so is not a part of their day jobs.
```

Comment:
We might be able to do more with the Pull Request template, e.g. a checklist asking if they've tested their changes, etc?

Reply:
I agree, I think we need to ask them what it way did it work before and what way it works now with their new changes. So if we are approving a PR, we know where to look for their changes and so on.

Reply:
I've added a checklist with those items.